### DISCUSSION TITLE
Because IB Gateway is needed...

### DISCUSSION THREAD

**Discussion by datasprings** [stensoosaar/IBKit#2] [General]
Does this many any Xcode development that isn't on an actual Mac would not be possible? Such as building out API integration with apps for iPhone or iVisionOS?

**Comment by stensoosaar** [comment]
In these days IB provides also [web based api](https://ibkrcampus.com/ibkr-api-page/cpapi/#endpoints) which might be more suitable for development for other devices than mac. Alternatively you can host your personal ib gateway on remote computer and steam data to end user device, but i guess it's hardly worth of hassle..
